ckanext/dcatapchharvest/tests/test_dcatap_ch_parse.py

```diff
@@ -155,6 +155,13 @@ def test_dataset_all_fields(self):
             sorted(resource['documentation']),
             ['https://example.com/documentation-distribution-1', 'https://example.com/documentation-distribution-2']
         )
+        eq_(
+            sorted(resource['access_services']),
+            [
+                "https://example.com/my-great-data-service-1",
```

Comment:
please use single ` ' ` 
for consistency

Reply:
fixed :+1:

Reply:
should we also make an unicode string here, as in all other strings

Reply:
it doesn't make a difference to the testing (because `u"foo" == "foo" = True`), but I added it for consistent style :)